sys/net/vnet.h

```diff
@@ -287,8 +287,21 @@ extern struct sx vnet_sxlock;
 #define	VNET_DEFINE_STATIC(t, n) \
     static t VNET_NAME(n) __section(VNET_SETNAME) __used
 #endif
+#ifdef __CHERI_PURE_CAPABILITY__
+/*
+ * XXX: This cannot use exact bounds currently as vnet variables are
+ * not suitably aligned.  In particular, V_ip6qb in
+ * sys/netinet6/frag6.c is a large variable requiring larger
+ * alignment.
```

Comment:
This should probably mention _why_ they're not aligned, namely the toolchain is deficient and doesn't align+pad (or let you align+pad) variables with section attributes, and that once that's fixed we should be able to set exact bounds.

Reply:
Or possibly this should link to the issue directly even?  It could use updating certainly.